README.md

```diff
@@ -50,13 +50,17 @@ pip install astro-provider-databricks
 4. [Create using your preferred way](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) a Databricks Airflow connection (so Airflow can access Databricks using your credentials). This can be done by running the following command, replacing the login and password (with your access token):
 
 ```shell
+# If using Airflow 2.3 or higher:
 airflow connections add 'databricks_conn' \
     --conn-json '{
         "conn_type": "databricks",
         "login": "some.email@yourcompany.com",
         "host": "https://dbc-c9390870-65ef.cloud.databricks.com/",
         "password": "personal-access-token"
     }'
+
+# If using Airflow 2.2:
+airflow connections add 'databricks_conn' --conn-type 'databricks' --conn-login 'some.email@yourcompany.com' --conn-host '[https://dbc-9c390870-65ef.cloud.databricks.com/' --conn-password 'personal-access-token'
```

Comment:
The `conn-host` has a leading square bracket `[` to indicate a list, but no closing square bracket `]`. Can we check once the right value, please?